content/en/blog/2024/gc-elections.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Announcing the 2024 OpenTelemetry Governance Committee Election
+linkTitle: 2024 GC Election
+date: 2024-09-03
+author: OpenTelemetry Governance Committee
+aliases: [gc-elections-2024]
```

Comment:
FYI, this alias isn't needed, because the file was never published as `gc-elections-2024`. I'll submit a followup PR so that we don't continue to propagate this unnecessary alias each year 😄.

Reply:
Thanks! I didn't know alias was not a required field. Although it does make sense.

Reply:
Right, not obvious -- no worries. See #5145 for the followup.